.github/workflows/bluetooth.yaml

```
@@ -32,6 +24,10 @@ jobs:
       BSIM_COMPONENTS_PATH: /opt/bsim/components
       EDTT_PATH: ../tools/edtt
       bsim_bt_test_results_file: ./bsim_bt_out/bsim_results.xml
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

As identified by @PerMac, please change this to use `head_ref` instead of `ref`, since the latter is always set to the base branch (e.g. main) for the `pull_request_target` events.

```suggestion
      group: ${{ github.workflow }}-${{ github.head_ref }}
```

Similar changes need to be made for the rest.